test/testenv/secretutil.go

```diff
@@ -28,6 +28,13 @@ var SecretKeytoServerConfStanza = map[string]string{
 	"shc_secret":   "shclustering",
 	"idxc_secret":  "clustering",
 	"pass4SymmKey": "general",
+	"hec_token":    "http://splunk_hec_token",
+}
+
+// SecretAPIRequests map to convert secretName to respective api requests
+var SecretAPIRequests = map[string]string{
```

Comment:
It will be better to make these formatted strings. That way call can change  `user`, `password`, `host`, `port`.

Reply:
modified